extensions/amp-gwd-animation/0.1/amp-gwd-animation.js

```diff
@@ -241,6 +241,7 @@ export class GwdAnimation extends AMP.BaseElement {
     const eventName = getDetail(event)['eventName'];
     const timelineEventName = `${this.timelineEventPrefix_}${eventName}`;
 
+    // TODO(wg-ui-and-a11y): Should animation timeline events be low trust?
```

Comment:
@ampproject/wg-ui-and-a11y

Reply:
We talked about this some this week--the component is externally contributed and not documented on amp.dev, so we are looking into current usage.

Reply:
Turns out (1) there is usage, just none detected on web because (2) this is an Ads component. Over to @ampproject/wg-ads